src/tribol/mesh/MfemData.hpp

```diff
@@ -988,6 +990,13 @@ class MfemMeshData {
     return submesh_lor_xfer_ ? submesh_lor_xfer_->GetLORGridFn().ParFESpace() : nullptr;
   }
 
+  /**
+   * @brief Get the number of element subdivisions per dimension on the LOR mesh
```

Comment:
Minor, but the number of subdivisions is equal in all directions across the HO face, right?

Reply:
Correct